Edge/Integrations/Analytics/Adobe/ListVar/listvar.projectjs.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Adobe Analytics Integration
+ * [Installed in Project JS]
+ */
+var adobeIntegrator = {
+  LISTVAR: "YOUR-LISTVAR-HERE, ie: 'list3'",
+  campaignArray: [],
+  assignCampaigns: function(sVariable) {
+    sVariable[this.LISTVAR] = sVariable[this.LISTVAR] || [];
+    for (var i = this.campaignArray.length - 1; i >= 0; i--) {
+      sVariable[this.LISTVAR].push(this.campaignArray[i]);
+      this.campaignArray.splice(i, 1);
```

Comment:
nit: You could use other array operators like `.pop` that return an item and also mutate the list

```
var lastElement = this.campaignArray.pop()
sVariable[this.LISTVAR].push(lastElement)
```